doc/source/ray-core/tasks.rst

```diff
@@ -14,7 +14,7 @@ Ray enables arbitrary functions to be executed asynchronously on separate Python
             :start-after: __tasks_start__
             :end-before: __tasks_end__
 
-        See the `ray.remote package reference <package-ref.html>`__ page for specific documentation on how to use ``ray.remote``.
+        See the :ref: `ray.remote package reference <ray.remote>` page for specific documentation on how to use ``ray.remote``.
```

Comment:
```suggestion
        See the :ref:`ray.remote package reference <ray.remote>` page for specific documentation on how to use ``ray.remote``.
```

<img width="840" alt="image" src="https://github.com/ray-project/ray/assets/3967392/0985a73a-c76d-4fd8-b50c-fd34f00d0dbc">

Reply:
Thank you for catching that. 🤦‍♀️